<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleSurveyGui</class>
 <widget class="QWidget" name="SimpleSurveyGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Survey</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="7" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLineEdit" name="start_distance">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="start_bearing"/>
         </item>
        </layout>
       </item>
       <item row="6" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLineEdit" name="start_easting">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="start_northing">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="11" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLineEdit" name="mark_easting">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mark_northing">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Latitude, Longitude</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>From Start</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Distance, Bearing</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Distance, Bearing</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Position</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="heading_disp"/>
         </item>
         <item>
          <widget class="QLineEdit" name="altitude_disp"/>
         </item>
        </layout>
       </item>
       <item row="13" column="1">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Elevation, slope</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QLineEdit" name="latitude_disp">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="longitude_disp">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Position</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Elevation, slope</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>GPS Current</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="3">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>From Mark</string>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLineEdit" name="mark_distance"/>
         </item>
         <item>
          <widget class="QLineEdit" name="mark_bearing">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="13" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLineEdit" name="mark_elevation">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mark_slope"/>
         </item>
        </layout>
       </item>
       <item row="8" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLineEdit" name="start_elevation">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="start_slope"/>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="source_label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Source (&lt;a href=&quot;/refresh&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;refresh&lt;/span&gt;&lt;/a&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextBrowserInteraction</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="gps_source_pick">
         <item>
          <property name="text">
           <string>System location services</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Simulated from log file</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Connect to server</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Heading, ellipsoidal height</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="units_metres">
         <property name="text">
          <string>Metres</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">units_button_group</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="units_feet">
         <property name="text">
          <string>Feet and Inches</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">units_button_group</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="quit_button">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="start_button">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mark_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Mark</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>start_button</sender>
   <signal>clicked()</signal>
   <receiver>mark_easting</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_button</sender>
   <signal>clicked()</signal>
   <receiver>mark_northing</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_button</sender>
   <signal>clicked()</signal>
   <receiver>mark_distance</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_button</sender>
   <signal>clicked()</signal>
   <receiver>mark_bearing</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quit_button</sender>
   <signal>clicked()</signal>
   <receiver>SimpleSurveyGui</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="units_button_group"/>
 </buttongroups>
</ui>
